sites/www/changelog.rst

```diff
@@ -246,7 +246,7 @@ Changelog
     required if your target systems do not support either RSA2 or the
     ``server-sig-algs`` protocol extension.
 
-    Specifically, you need to specify ``disabled_algorithms={'keys':
+    Specifically, you need to specify ``disabled_algorithms={'pubkeys':
```

Comment:
Is `pubkeys` the correct key to use in `disabled_algorithms` in *all* cases? Or is the behavior more complex than that? (Honest question, I'm not deeply familiar.)

Reply:
It's always hard to tell about *all* cases so the short answer is "I am not sure", but many things make me believe this was a typo.

This changelog entry has a more detailed explanation below and here it is mentioned to use `pubkeys`:

https://github.com/paramiko/paramiko/blob/4bf5d30e0dd6eb7ebbecc03445c3b77f61ad761d/sites/www/changelog.rst?plain=1#L275-L276

Using `pubkeys` was the solution for  https://github.com/paramiko/paramiko/issues/2090#issuecomment-1192829965 and also for me when trying to connect to a "XFB.Gateway" server.

Also, this is documented as a workardound for servers not supporting the [`server-sig-algs`]( https://www.rfc-editor.org/rfc/rfc8308#section-3.1) extension and this is an extension for public key algorithms.

`keys` was added in https://github.com/paramiko/paramiko/commit/5f222495b5a62f3a1c465292bcace15888f40515 maybe @bitprophet can confirm ? Thanks !